tensorboard/plugins/histogram/BUILD

```diff
@@ -125,6 +125,28 @@ py_test(
     ],
 )
 
+py_test(
+    name = "large_count_summary_test",
+    # Ensure enough RAM for `test_with_large_counts`.
+    size = "large",
+    srcs = ["summary_test.py"],
+    args = [
+        "SummaryV1OpTest.test_with_large_counts",
```

Comment:
Oh right, I forgot this is actually a bunch of individual tests. Perhaps we can instead use `args = ["-k", "test_with_large_counts"]`?

https://docs.python.org/3.7/library/unittest.html#cmdoption-unittest-k

Reply:
Thanks for the suggestion. The first arg needs to be `"-- -k"` for some reason otherwise the flag won't be recognized.